flipperdevices/flipperzero-firmware
# [FL-2998] IR TV Universal Remote refactor and docs

# What's new

- Renamed the signals in the same style as in Audio and A/C universal remotes.
- Added docs on adding TVs to the universal remote.

# Verification 

- Checkout, compile, flash (the uj)
- Copy the `assets/resources/infrared/assets/tv.ir` file to `/ext/infrared/assets`
- Go to `Infrared -> Universal Remotes -> TVs`
- Send some signals! The signals should be sent as before.

# Checklist (For Reviewer)
